lib/test/api/InjectBackendClient.js

```diff
@@ -1,36 +1,43 @@
-import BackendClient from '@/lib/api/BackendClient';
+import QueryString from '@/lib/api/QueryString';
```

Comment:
we should probably test `InjectBackendClient` as well, similar to our unit tests for `AxiosBackendClient`